src/js/messaging/containers/Compose.jsx

```diff
@@ -22,6 +33,10 @@ class Compose extends React.Component {
     this.props.setSubject(valueObj);
   }
 
+  dispatchRecipientChange(event) {
```

Comment:
Standardize on the argument name with the other `dispatch*` functions.

Although, on that topic, I'm not sure if `dispatch*` is the right name. Like for `dispatchCategoryChange`, it's actually calling two actions that have been bound to the dispatch. Perhaps `handle*`?